internal/backends/winit/renderer/femtovg/glcontext.rs

```diff
@@ -54,7 +57,7 @@ impl OpenGLContext {
     pub fn new_context<T>(
         window_builder: winit::window::WindowBuilder,
         window_target: &winit::event_loop::EventLoopWindowTarget<T>,
-    ) -> Result<(winit::window::Window, Self), PlatformError> {
+    ) -> Result<(Rc<winit::window::Window>, Self), PlatformError> {
```

Comment:
I think this is the right thing to do because moving to raw-window-handle 0.6 will also need us to use Rc there (my early patch did that)